test/demo/index.html

```diff
@@ -1151,6 +1164,23 @@ <h3>Timeseries</h3>
         })
         .render();
 
+        window.metricComboChart = new KeenDataviz({
+          container: '.chart-metric-combo',
+          title: 'Metric Combo',
+          type: 'metric-combo',
+          metricResults: {
+            previous: metricComboChartData.previous,
+            current: metricComboChartData.current,
+          },
+          results: metricComboChartData.interval,
+          tooltip: {
+            format: {
+              title: (d) => new Date(d).toLocaleString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true })
+            }
+          }
+        })
+        .render();
```

Comment:
please add this chart type to the list
https://github.com/keen/keen-dataviz.js/tree/master/docs#custom-built-by-us
between metric and message